apimodels/perf.go

```diff
@@ -19,32 +20,37 @@ type GetCedarPerfCountOptions struct {
 	Execution int    `json:"_"`
 }
 
-func (opts GetCedarPerfCountOptions) convert() perf.GetOptions {
-	return perf.GetOptions{
-		Cedar: timber.GetOptions{
-			BaseURL: fmt.Sprintf("https://%s", opts.BaseURL),
-		},
-		TaskID:    opts.TaskID,
-		Execution: utility.ToIntPtr(opts.Execution),
-		Count:     true,
-	}
-}
-
 // CedarPerfCount holds one element, NumberOfResults, matching the json returned by
 // Cedar's perf count rest route.
 type CedarPerfCount struct {
 	NumberOfResults int `json:"number_of_results"`
 }
 
-// CedarPerfResultsCount queries Cedar for the number of perf results attached to a task.
+// CedarPerfResultsCount queries SPS for the number of perf results attached to a task.
 func CedarPerfResultsCount(ctx context.Context, opts GetCedarPerfCountOptions) (*CedarPerfCount, error) {
```

Comment:
Apologies for the nit, could we rename this function and related structs used here to not reference cedar, since we aren't calling cedar?